SocialCareCaseViewerApi/V1/Gateways/ProcessDataGateway.cs

```diff
@@ -138,6 +140,17 @@ public ProcessDataGateway(ISccvDbContext sccvDbContext)
             return new Tuple<IEnumerable<CareCaseData>, int>(response, totalCount);
         }
 
+        public CareCaseData GetCaseById(string recordId)
+        {
+            var collection = _sccvDbContext.getCollection();
+            var filter = Builders<BsonDocument>.Filter.Eq("_id", ObjectId.Parse(recordId));
```

Comment:
this will throw an exception if the string is not in a valid format (can be handled in boundary object validation)

Reply:
boundary object validation added